modules/twinklexfd.js

```diff
@@ -24,12 +24,42 @@ Twinkle.xfd = function twinklexfd() {
 	Twinkle.addPortletLink(Twinkle.xfd.callback, 'XFD', 'tw-xfd', 'Start a deletion discussion');
 };
 
-Twinkle.xfd.num2order = function twinklexfdNum2order(num) {
-	switch (num) {
-		case 1: return '';
-		case 2: return '2nd';
-		case 3: return '3rd';
-		default: return num + 'th';
+
+var utils = {
+	/** Get ordinal number figure */
+	num2order: function(num) {
+		switch (num) {
+			case 1: return '';
+			case 2: return '2nd';
+			case 3: return '3rd';
+			default: return num + 'th';
+		}
+	},
+
+	/**
+	 * Remove namespace name from title if present
+	 * Exception-safe wrapper around mw.Title
+	 * @param {string} title
+	 */
+	stripNs: function(title) {
+		var title_obj = new mw.Title(title);
```

Comment:
Just a quick comment before I merge the underlying commits, but I believe this needs to be `mw.Title.newFromText` to actually be safe.